glide.yaml

```diff
@@ -25,6 +25,7 @@ import:
   - libmachine/state
   - libmachine/swarm
 - package: github.com/google/go-github
+  version: 30a21ee1a3839fb4a408efe331f226b73faac379
```

Comment:
The latest commit of `google/go-github` is not working due to no of arguments change in few functions.